syne_tune/optimizer/schedulers/multiobjective/moasha.py

```diff
@@ -93,7 +93,7 @@ def __init__(
                     m in ["min", "max"] for m in mode
                 ), "all modes should be 'min' or 'max'."
             else:
-                assert mode in ["min", "max"], "`mode` must be 'min' or 'max'."
+                assert mode in ["min", "max"], "``mode`` must be 'min' or 'max'."
```

Comment:
Intentional or just caught by the mass find-replace?

Reply:
Intentional. Should do the same in messages and doc strings